### ISSUE TITLE
Linter Node : no linters matched query

### ISSUE THREAD

**Issue by hamelg** [derailed/popeye#341]
**Describe the bug**
The linter Node is never activated 

**To Reproduce**

`popeye -s no -A`

says

`Boom! 💥 no linters matched query. check section selector`

**Debug output**
```
2:29PM DBG Connecting to API Server https://api.xxxxr:6443
2:29PM DBG Lint 7.743575ms
2:29PM ERR 💥 no linters matched query. check section selector

2:29PM ERR goroutine 1 [running]:
runtime/debug.Stack()
	runtime/debug/stack.go:26 +0x5e
github.com/derailed/popeye/pkg.BailOut({0x5db138a666c8, 0xc0007e47e0})
	github.com/derailed/popeye/pkg/helpers.go:24 +0x10f
github.com/derailed/popeye/cmd.doIt.func1()
	github.com/derailed/popeye/cmd/root.go:57 +0x7e
panic({0x5db1385bc9e0?, 0xc0007e47e0?})
	runtime/panic.go:785 +0x132
github.com/derailed/popeye/cmd.bomb({0x5db138a666c8?, 0xc0007e4730?})
	github.com/derailed/popeye/cmd/root.go:87 +0xdb
github.com/derailed/popeye/cmd.doIt(0xc000121100?, {0x5db137d9cac5?, 0x4?, 0x5db137d9c991?})
	github.com/derailed/popeye/cmd/root.go:73 +0xf0
github.com/spf13/cobra.(*Command).execute(0x5db13a253720, {0xc000052110, 0x3, 0x3})
	github.com/spf13/cobra@v1.8.1/command.go:989 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0x5db13a253720)
	github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/derailed/popeye/cmd.Execute(...)
	github.com/derailed/popeye/cmd/root.go:48
main.main()
	github.com/derailed/popeye/main.go:28 +0x1b
```

**Expected behavior**
The list of node lints must be displayed.

**Versions (please complete the following information):**
 - OS: openshift 4.14
 - Popeye 0.21.5
 - K8s 1.27

**Comment by derailed** [comment]
@hamelg Good catch! Thank you for the heads up.

**Comment by derailed** [comment]
Fixed v0.21.6
